command.c

```diff
@@ -5,11 +5,9 @@
 
 #include "nsq.h"
 
-const static char * NEW_LINE  = "\n";
-const static char * NEW_SPACE = " ";
-const static int MAX_BUF_SIZE = 128;
-const static int MIN_BUF_SIZE = 64;
-const static int BUF_DELTER   = 2;
+static const int MAX_BUF_SIZE = 128;
+static const int MIN_BUF_SIZE = 64;
+static const int BUF_DELTER   = 2;
```

Comment:
unsigned int declaration should be better

Reply:
These could change to `size_t`, which is what they are compared to.

... but the logic that uses these functions is actually quite wrong and needs a lot more replacing ... e.g. `buf_size` in `nsq_buffer_add()` is never updated, and `if (buf_size - n < MIN_BUF_SIZE || buf_size - l < MIN_BUF_SIZE)` should be `if (buf_size - n < l + MIN_BUF_SIZE)`. I was planning to tackle this in the next pull request :)

Reply:
ok, understand, let me fix this.

Reply:
OK, thanks.